client/src/models/businessConstants.ts

```diff
@@ -0,0 +1,723 @@
+import _ from "lodash";
+
+import { assign_to_dev_helper_namespace } from "src/core/assign_to_dev_helper_namespace";
+import { lang } from "src/core/injected_build_constants";
+
+interface BusinessConceptTextValue {
+  text: string;
+}
+interface BilingualBusinessConceptTextValue {
+  en: string;
+  fr: string;
+}
+
+type RawBusinessConceptInputTextValue =
+  | BilingualBusinessConceptTextValue
+  | BusinessConceptTextValue;
+
+function getNormalizedEntry(
+  obj: RawBusinessConceptInputTextValue
+): BusinessConceptTextValue {
+  if (obj.hasOwnProperty("text")) {
+    return { text: (obj as BusinessConceptTextValue).text };
+  } else if (obj.hasOwnProperty(lang)) {
+    return { text: (obj as BilingualBusinessConceptTextValue)[lang] };
+  } else {
+    throw "shouldn't happen";
+  }
+}
+
+interface RawBusinessConceptEntrySet {
+  [key: string]: RawBusinessConceptInputTextValue;
+}
+interface BusinessConceptEntrySet {
+  [key: string]: BusinessConceptTextValue;
+}
+
+function normalizeBilinguals(
+  entries: RawBusinessConceptEntrySet
+): BusinessConceptEntrySet {
+  return _.mapValues(entries, (entry) => getNormalizedEntry(entry));
+}
+
```

Comment:
Everything outside this block is basically noise. What's going on here is that we have two ways of defining a business constant, either bilingual `{en, fr}` dicts or `{text}` dicts. This code deals with converting both types to `{text}` dicts.

Note the use of a common TS pattern of checking properties via `hasOwnProperty` and the use of the `as` keyword.

There's a good chance there's a cleaner pattern I could have used here. [these docs](https://www.typescriptlang.org/docs/handbook/2/narrowing.html) are relevant. This PR is a bit of a conflict magnet, though, so I'd say we can explore this internal refactor after merging.